static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -407,7 +407,9 @@ define([
     }
 
     function shouldRenderLabel($adSlot) {
-        return $adSlot.data('label') !== false && qwery('.ad-slot__label', $adSlot[0]).length === 0;
+        return !$adSlot.hasClass('ad-slot--frame') &&
```

Comment:
I can see a refactoring of our code where every creative is confined to its own module and deals with everything that is its own business ... indeed this has nothing to do in dfp-api.